Source/ZoomNet/Resources/IWebinars.cs

```diff
@@ -51,11 +51,12 @@ public interface IWebinars
 		/// <param name="settings">Webinar settings.</param>
 		/// <param name="trackingFields">Tracking fields.</param>
 		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <param name="templateId">Template Identifer.</param>
 		/// <returns>
 		/// The new webinar.
 		/// </returns>
 		/// <exception cref="System.Exception">Thrown when an exception occured while creating the webinar.</exception>
-		Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string topic, string agenda, DateTime start, int duration, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default);
+		Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string topic, string agenda, DateTime start, int duration, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default, string templateId = null);
```

Comment:
Generally speaking it's good practice to add new parameters at the end, as you have done, but there is one notable exception to this rule (at least in my opinion): the CancellationToken should always be the last parameter in an async method signature. So, please position the new templateId parameter before the cancellationToken.

Reply:
Great suggestion - I also agree (and should have spotted this). I've updated the code.